cumulusci/core/flowrunner.py

```diff
@@ -415,7 +415,7 @@ def run(self, org_config):
             for step in self.steps:
                 self._run_step(step)
             flow_name = f"'{self.name}' " if self.name else ""
-            self.logger.info(f"Completed flow {flow_name}successfully!")
+            self.logger.info(f"Completed flow {flow_name} on org {org_config.name}successfully!")
```

Comment:
Notice from the line above that flow_name may either be the name with a space afterward, or an empty string if this flow has no name. So for the spacing to work out right I think we need:

```suggestion
            self.logger.info(f"Completed flow {flow_name}on org {org_config.name} successfully!")
```

Reply:
@davisagli ah thanks updated